deps/chakrashim/lib/chakra_debug.js

```diff
@@ -0,0 +1,1304 @@
+          request.arguments.stepaction);
+      }
+
+      if (!CallHostFunction(chakraDebug.JsDiagSetStepType,
+        jsDiagSetStepType)) {
+        success = false;
+      }
+    }
+
+    response.success = success;
+
+    // We are continuing, delete the globalExecutionState
+    globalExecutionState = undefined;
+  };
+
+  V8CommandProcessor.prototype.setbreakpoint = function(request, response,
+    optAddToPending) {
+    var breakpointType;
+    var breakpointScript;
+    var breakpointTarget;
+    var line = 0;
+    var column = 0;
+
+    if (request.arguments) {
+      var scriptObject;
+      if (request.arguments.type == 'scriptRegExp') {
+        var targetRegex = new RegExp(request.arguments.target);
+        scriptObject =
+          DebugManager.ScriptsManager.GetScriptFromMatchingFileName(
+          targetRegex);
+        if (scriptObject) {
+          breakpointScript = scriptObject;
+          breakpointType = Debug.ScriptBreakPointType.ScriptRegExp;
+          breakpointTarget = request.arguments.target;
+        }
+      } else if (request.arguments.type == 'script' &&
+        (request.arguments.target == null ||
+        request.arguments.target == undefined)) {
+        var breakScriptId = globalExecutionState.GetBreakScriptId();
+        scriptObject = DebugManager.ScriptsManager.GetScript(breakScriptId);
+        if (scriptObject) {
+          breakpointScript = scriptObject;
+          breakpointType = Debug.ScriptBreakPointType.ScriptId;
+          breakpointTarget = request.arguments.target;
+        }
+      } else if (request.arguments.type == 'script' &&
+        request.arguments.target) {
+        scriptObject = DebugManager.ScriptsManager.GetScriptFromFileName(
+          request.arguments.target);
+        if (scriptObject) {
+          breakpointScript = scriptObject;
+          breakpointType = Debug.ScriptBreakPointType.ScriptName;
+          breakpointTarget = request.arguments.target;
+        }
+      } else if (request.arguments.type == 'scriptId' &&
+        request.arguments.target) {
+        var scriptId = parseInt(request.arguments.target);
+        scriptObject = DebugManager.ScriptsManager.GetScript(scriptId);
+        if (scriptObject) {
+          breakpointScript = scriptObject;
+          breakpointType = Debug.ScriptBreakPointType.ScriptId;
+          breakpointTarget = request.arguments.target;
+        }
+      }
+
+      if (typeof request.arguments.line == 'number') {
+        line = request.arguments.line;
+      }
+
+      if (typeof request.arguments.column == 'number') {
+        column = request.arguments.column;
+      }
+    }
+
+    if (breakpointScript) {
+      var v8Breakpoint = new V8Breakpoint(breakpointType, breakpointTarget,
+        breakpointScript, line, column);
+      var success = false;
+      if (v8Breakpoint.Set()) {
+        success = true;
+        DebugManager.BreakpointManager.AddBreakpoint(v8Breakpoint);
+      }
+      // ToDo: Add to list
```

Comment:
No, seems a left over. Removed

---
In reply to: [82074541](https://github.com/nodejs/node-chakracore/pull/126#discussion_r82074541) [](ancestors = 82074541)